src/components/cards/TaskCard.tsx

```diff
@@ -37,8 +39,11 @@ export const TaskCard = ({ task }: { task: TaskResponse }) => {
           <Typography variant="sm" fontSize="12px" sx={{ color: (theme) => theme.color.gray[500] }}>
             {(currentAssignee as IAssigneeCombined).name === 'No assignee'
               ? 'No assignee'
-              : (currentAssignee as IAssigneeCombined)?.name ||
-                `${(currentAssignee as IAssigneeCombined)?.givenName ?? ''} ${(currentAssignee as IAssigneeCombined)?.familyName ?? ''}`.trim()}
+              : truncateText(
+                  (currentAssignee as IAssigneeCombined)?.name ||
+                    `${(currentAssignee as IAssigneeCombined)?.givenName ?? ''} ${(currentAssignee as IAssigneeCombined)?.familyName ?? ''}`.trim(),
+                  TruncateMaxNumber.TASK_CARD_NAME,
+                )}
```

Comment:
I think this computation is unnecessary overload to the app. I think it is better to create an `hoc` that supports trucating with the styling below. We can achieve this using simple css styling - 
```css
width: 146px; //the size of visible element
overflow: hidden;
text-overflow: ellipsis;
white-space: nowrap;
```